Packages/com.vrchat.ClientSim/package.json

```diff
@@ -14,23 +14,23 @@
 		"com.unity.inputsystem" : "1.0.2"
 	},
 	"gitDependencies" : {
-		"com.vrchat.base" : "https://github.com/vrchat/packages.git?path=/packages/com.vrchat.base#3.0.2",
-		"com.vrchat.worlds" : "https://github.com/vrchat/packages.git?path=/packages/com.vrchat.worlds#3.0.2"
+		"com.vrchat.base" : "https://github.com/vrchat/packages.git?path=/packages/com.vrchat.base#3.0.4",
+		"com.vrchat.worlds" : "https://github.com/vrchat/packages.git?path=/packages/com.vrchat.worlds#3.0.4"
 	},
-	"url" : "https://github.com/vrchat-community/ClientSim.git?path=/Packages/com.vrchat.ClientSim#1.1.3",
+	"url" : "https://github.com/vrchat-community/ClientSim.git?path=/Packages/com.vrchat.ClientSim#1.1.4",
 	"samples" : [
 		{
 			"displayName" : "ClientSimWorldTestExample",
 			"description" : "An example of a world test using the Test Framework included in ClientSim. Automate interactions in your world and verify the results. Currently requires knowledge of C# and Unity Testing.",
 			"path" : "Samples~/ClientSimWorldTestExample"
 		},
 		{
-			"displayName" : "ClientSimIssue3RespawnTest",
-			"description" : "A  world test to reproduce the ClientSim Respawn issue.",
-			"path" : "Samples~/ClientSimIssue3RespawnTest"
+			"displayName" : "ClientSimSelfTests",
+			"description" : "Tests to ensure ClientSim works as expected.\r\n* ClientSim Respawn issue",
```

Comment:
Huh, and the new lines work properly in the package manager?

Reply:
It doesn't actually display anywhere in the Unity 2019 Package Manager as far as I can see, so I updated it simply for correctness.

Reply:
Ah, I was mistaking it for the package description instead of the sample description. I'm curious why you added the new line `\r\n` if it is never displayed. My impression is that json would keep the actual characters instead of converting to the special character. Overall it doesn't matter if no one ever sees it.